### ISSUE TITLE
Null reference exception when clicking on manipulation handler with gaze pointer

### ISSUE THREAD

**Issue by luis-valverde-ms** [Microsoft/MixedRealityToolkit-Unity#4117]
1. Open HandInteractionExamples scene
2. Play scene with an Xbox controller connected
3. Look at an object with a manipulation handler and press the A button in the controller
4. Observe following error:

```
NullReferenceException: Object reference not set to an instance of an object
Microsoft.MixedReality.Toolkit.UI.ManipulationHandler.GetPointersVelocity () (at Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs:232)
```

This is happening because manipulation handler assumes that all pointers have a valid controller, which is not the case for the gaze pointer.

**Comment by StephenHodgson** [comment]
You're right, not all pointers have controllers, but they all do have valid `InputSources`

**Comment by luis-valverde-ms** [comment]
Fixed by #4118
